package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "starkexpress-dotnet-client",
+  "version": "0.0.0",
+  "description": "[gha]: https://github.com/threesigmaxyz/starkexpress-dotnet-client/actions [gha-badge]: https://github.com/threesigmaxyz/starkexpress-dotnet-client/actions/workflows/main.yml/badge.svg [dotnet]: https://github.com/dotnet/core [dotnet-badge]: https://img.shields.io/static/v1?label=Built%20with&message=.NET&color=blueviolet [license]: https://opensource.org/licenses/MIT [license-badge]: https://img.shields.io/badge/License-MIT-blue.svg",
```

Comment:
I understand this is acting as a placeholder but can we change it to a more descriptive value?

Reply:
Changed